l10n_ch_base_bank/models/bank.py

```diff
@@ -239,11 +239,16 @@ def _check_postal_num(self):
     def _get_acc_name(self):
         """ Return an account name for a bank account
         to use with a ccp for ISR.
-        This method make sure to generate a unique name
+        This method makes sure to generate a unique name
         """
+
         part_name = self.partner_id.name
+        if not part_name and self.env.context.get('default_partner_id'):
+            partner_id = self.env.context.get('default_partner_id')
```

Comment:
I don't see the `default_partner_id` being set in the `res.partner.bank` view.

While I like the idea, is it a customization on your side? How is odoo defining that `default_partner_id` ?

Reply:
Hi @yvaucher, thanks for the reply.

The path I use to create/access partner bank accounts is located in a partner form view (_Invoicing_ > _Sales_ > _Master Data_ > _Customers_ > open a customer > tab _Sales & Purchases_ > _Bank Account(s)_)

`default_partner_id` is set in `odoo/addons/account/views/partner_views.xml`, around line 124 from what I can see :

```
                <group name="container_row_2" position="inside">
                    <group string="Payments" name="banks">
                        <field name="currency_id" invisible="1"/>
                        <div>
                            <button type="action" class="btn-link"
                                name="%(base.action_res_partner_bank_account_form)d"
                                context="{'search_default_partner_id': active_id, 'default_partner_id': active_id}">
                                <field string="Bank account(s)" name="bank_account_count" widget="statinfo"/>
                            </button>
                        </div>

                    </group>
                </group>
```